### PR TITLE
Implementation of multi-language support for backend

### PR DESCRIPTION
**Pull request by alena456**

### CODE REVIEW

**Review by mahvaezi — CHANGES_REQUESTED** [review]
Could you please add "language_code" to the output of the endpoint "participant" in openapi.yaml?

File: src/assets/openapi.yaml [hunk 1]
```diff
@@ -219,6 +242,26 @@ paths:
                     $ref: '#/components/responses/500ServerError'
             security:
                 - Bearer: []
+    /participant/update-language-code/{subjectId}:
```
[the remark this answers is not in the document]
Comment (alena456) [reply src/assets/openapi.yaml:245]: Hi, what do you mean by that? The api works just like "/participant/{subjectId}" and has the language code in the request body

File: db/flyway.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ module.exports = function() {
     return {
         flywayArgs: {
             // JDBC string that addresses the database
-            url: 'jdbc:postgresql://localhost:15432/compass',
+            url: 'jdbc:postgresql://localhost:5432/compass',
```
[the remark this answers is not in the document]
Comment (alena456) [reply db/flyway.js:6]: I will revert.

**Comment by alena456** [comment]
> language_code

Updated in latest commit

**Comment by mahvaezi** [comment]
@alena456 it would be great if the endpoint "participant" would send information about the language of the study participant. Could you still implement this change considering the time?

**Comment by mahvaezi** [comment]
> > language_code
> 
> Updated in latest commit

You have to adjust the function too.

**Comment by alena456** [comment]
> language_code

Should be working now

**Review by mahvaezi — APPROVED** [review]

**Comment by alena456** [comment]
> Is there any specific reason why DB is not exported as default anymore?

@lenzch Can you answer that question? These changes were already present in your initial work on the multi-language change.

**Comment by lenzch** [comment]
> Is there any specific reason why DB is not exported as default anymore?

@mbastian93 The export was changed to better comply with the way the exports are used in other parts of the application.

**Comment by mbastian93** [comment]
It seems like there are some lint/prettier issues and the tests also fail when run locally because the participant entry misses the `language_code`. Did you somehow deactivate the husky pre-commit hook?

**Comment by alena456** [comment]
> It seems like there are some lint/prettier issues and the tests also fail when run locally because the participant entry misses the `language_code`. Did you somehow deactivate the husky pre-commit hook?

src/types/ParticipantEntry.ts  already has language_code for Participant. It was part of this commit: https://github.com/NUMde/compass-numapp-backend/pull/70/commits/0ff88f4b59ff5a08b750ec88fdd2857cfca97ecd. Do you mean this part with "participant entry"?
No change was made for pre-commit hook from my side.

**Comment by mbastian93** [comment]
> > It seems like there are some lint/prettier issues and the tests also fail when run locally because the participant entry misses the `language_code`. Did you somehow deactivate the husky pre-commit hook?
> 
> src/types/ParticipantEntry.ts already has language_code for Participant. It was part of this commit: [0ff88f4](https://github.com/NUMde/compass-numapp-backend/commit/0ff88f4b59ff5a08b750ec88fdd2857cfca97ecd). Do you mean this part with "participant entry"? No change was made for pre-commit hook from my side.

I'm talking about the [ExampleStateModel](https://github.com/alena456/compass-numapp-backend/blob/i18n/tests/models/ExampleStateModel.test.ts).

**Pull request merged**